Recording/Stages/SendFountainInfo.asm

```diff
@@ -0,0 +1,52 @@
+################################################################################
+# Address: 801cc998
+################################################################################
+.include "Common/Common.s"
+.include "Recording/Recording.s"
+
+################################################################################
+# Routine: SendFountainInfo
+# ------------------------------------------------------------------------------
+# Sends Fount of Dreams platform heights when they change.
+################################################################################
+
+.set REG_Buffer,4
+.set REG_BufferOffset,5
+
+# We skip to avoid the two initialization calls at game start
+  mflr r0
+  load r3, 0x801cc908
+  xor. r0, r0, r3
+  beq Skip
```

Comment:
```suggestion
# We skip to avoid the two initialization calls at game start
loadGlobalFrame r3
cmpwi r3, 1
ble Skip
```

Reply:
I think I'd prefer this over checking the LR, checking the LR feels quite hacky compared to checking the scene frame.

Reply:
Just tried this, but it doesn't work. The scene frame value is 0x000000d8.

Reply:
Ah you're right I was testing with a replay and the behavior is different. It looks like the function gets called in the previous scene. So 0xd8 was just the frame on the SSS that you selected FoD. We might be able to check if we're in an in-game scene instead... lemme check some stuff.

Reply:
Did you confirm that `FN_ShouldRecord` did not work? I feel like it might.

Reply:
So I'm thinking now that maybe the best plan is to add the following to `SendGameInfo.asm`
```
  li  r3,CONST_FirstFrameIdx
  stw r3,frameIndex(r13)
```
That way you know that the value has been initialized to the first frame and you can just use that.

Reply:
Is it really much safer to modify the scene frame counter? I agree checking the LR is hacky, but I can't think of a way it would break.

Reply:
Super late to this but that frameIndex is not the scene frame counter. That's Slippi's frame counter that starts at -123. Overwriting it there should be completely safe, nothing else in the game uses that.

Reply:
I did it this way and it worked.